### ISSUE TITLE
Multiclass logistic regression uses ovr instead of softmax by default

### ISSUE THREAD

**Issue by pavlin-policar** [biolab/orange3#4754]
**Describe the bug**
For some reason, the default Orange [LogisticRegression](https://github.com/biolab/orange3/blob/b5e910f4aef4b3e10daf997b494d51c2f220b236/Orange/classification/logistic_regression.py#L40) uses `multi_class='ovr'` instead of softmax. In [scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html), which this class wraps, the default value is set to `multi_class='auto'`, which 

> selects ‘ovr’ if the data is binary, or if solver=’liblinear’, and otherwise selects ‘multinomial’.

Our LogisticRegression also decides to changes the default solver to `liblinear`, even though the default in scikit-learn is `lbfgs`. 

To clarify, `multinomial` is softmax, and this is what we want.

**Orange version:**
master
